tools/customLinters/rubocop_pegasus_db_usage.rb

```diff
@@ -1,10 +1,10 @@
 module CustomCops
-  # Custom cop that checks for use of PEGASUS_DB outside of pegasus/ directory
+  # Custom cop that checks for use of pegasus DB outside of pegasus/ directory
   class PegasusDbUsage < RuboCop::Cop::Base
-    MSG = 'Do not use PEGASUS_DB outside of pegasus/ directory.'
+    MSG = 'Do not access pegasus DB from outside of the top-level pegasus/ directory.'
```

Comment:
How would you feel about including a link to https://github.com/code-dot-org/code-dot-org/pull/55417 in the message? I don't see this a lot in the code.org codebase, so perhaps that's a reason not to, but I personally like to include links to PRs when relevant in the code itself. Technically you can get this from git blame, but in actual practice PR links are much more likely to be followed and understood. No worries either way.

Reply:
happy to include it! sadly it has become routine to use git blame to find PR descriptions for clues in our repo, but that's certainly not a tradition I seek to preserve.